### PR TITLE
[PYT-567] Add an email subscribe form to Contributors page

### PR DESCRIPTION
**Pull request by jmjackson6880**
This PR adds a Salesforce form to the /resources/contributors page for email capture.

A few test submissions have been made. We will need @pytorchsam to confirm if they are working or not before we can move forward with merging this.

Preview: https://deploy-preview-676--frosty-bhabha-29c509.netlify.app/resources/contributors/

# Mobile Rendering

## On load
Screen Shot 2021-04-19 at 8 38 22 PM

## On email input
Screen Shot 2021-04-19 at 8 38 11 PM

## On submission
Screen Shot 2021-04-19 at 8 37 59 PM

# Desktop Rendering

## On load
Screen Shot 2021-04-19 at 8 32 12 PM

## On email input
Screen Shot 2021-04-19 at 8 35 08 PM

## On submission
Screen Shot 2021-04-19 at 8 37 46 PM

### CODE REVIEW

File: ecosystem/contributors.html [hunk 1]
```diff
@@ -35,27 +35,28 @@ <h2 class="lead">The central place for PyTorch contributors to stay up-to-date w
...
+      <div class="content-left">
+        <h2>Newsletter Sign Up</h2>
+        <p>Follow the contributors newsletter for curated news from across the PyTorch developer community</p>
+        <div class="contributor-form">
+          <form id="contributorForm" action="http://cl.s10.exct.net/DEManager.aspx" name="subscribeForm" method="post">
+            <input type="hidden" name="_clientID" value="515009167" />
+            <input type="hidden" name="_deExternalKey" value="BBAD707C-57C6-46F4-9701-D24E5550FB79" />
+            <input type="hidden" name="_action" value="add" />
+            <input type="hidden" name="_returnXML" value="0" />
+            <input type="hidden" name="_successURL" value="http://example.com/Success" />
```
Comment (suraj813) [inline ecosystem/contributors.html]: Can we replace these with valid URLs?

File: ecosystem/contributors.html [hunk 2]
```diff
@@ -35,27 +35,28 @@ <h2 class="lead">The central place for PyTorch contributors to stay up-to-date w
...
-                <button id ="submit" class="contributor-submit btn btn-outline-secondary" type="button">Submit</button>
-              </div>
-            </div>
-          </div> -->
+    <div class="two-column-row">
+      <div class="content-left">
+        <h2>Newsletter Sign Up</h2>
+        <p>Follow the contributors newsletter for curated news from across the PyTorch developer community</p>
+        <div class="contributor-form">
+          <form id="contributorForm" action="http://cl.s10.exct.net/DEManager.aspx" name="subscribeForm" method="post">
```
Comment (suraj813) [inline ecosystem/contributors.html]: This should be https

**Review by suraj813 — CHANGES_REQUESTED** [review]

File: ecosystem/contributors.html (hunk 1 above)
Comment (jmjackson6880) [reply ecosystem/contributors.html]: @suraj813 We had to switch to Mailchimp. This is outdated. Please see latest commit to this file.

File: ecosystem/contributors.html (hunk 2 above)
Comment (jmjackson6880) [reply ecosystem/contributors.html]: @suraj813 We had to switch to Mailchimp. This is outdated. Please see latest commit to this file.

**Pull request merged**